src/appendices/phenotype.md

```diff
@@ -0,0 +1,436 @@
+# Tabular phenotypic data guidelines
+
+This appendix is a collection of guidelines and examples
+for curating well-organized tabular phenotypic data.
+
+## Guidelines
+
+These guidelines are intended to improve the organization and clarity of
+tabular phenotypic data like the participants file, sessions file,
+and phenotypic and assessment data.
+
+They are recommendations and are by default ignored during validation.
+You can make them mandatory during validation by setting the
+[`AdditionalValidation` key](../modality-agnostic-files/dataset-description.md#additional-validation)
+contains `"Phenotype"` in the `dataset_description.json`.
+
+### 1. Aggregate data across sessions
+
+Aggregate participant information across all sessions into one tabular TSV file per
+measurement or phenotypic assessment and store this file in the `/phenotype` directory.
+Demographic information is a special case and  MUST be aggregated
+in the `participants.tsv` file at the root level of the dataset.
+It is RECOMMENDED to use the `age` column in the `participants.tsv` file
+to record participant age at every session in longitudinal or multi-session data sets.
+
+### 2. Always pair tabular data with data dictionaries
+
+Tabular phenotypic data MUST be prepared as one pair of a tabular file
+in tab-separated value (TSV) format and a corresponding data dictionary
+in JavaScript Object Notation (JSON) format.
+See the [Tabular files section](../common-principles.md#tabular-files) for more information.
+
+### 3. Add `MeasurementToolMetadata` to each tabular phenotypic measurement tool
+
+Whenever possible, it is RECOMMENDED to add `MeasurementToolMetadata` to
+each `phenotype/<measurement_tool_name>.json` data dictionary.
+This improves reusability and provides clarity about the measurement tool.
+See [`MeasurementToolMetadata` in the glossary](../glossary.md#measurementtoolmetadata-metadata) for more.
+
+### 4. Ensure minimal annotation for phenotypic and assessment data
+
+In phenotypic and assessment data, each measurement tool SHOULD have an independent
+aggregated data TSV file in which the user collects all subjects, sessions,
+and/or runs of data as one entry per row (with a row defined by
+the smallest unit of acquisition). In other words:
+
+-   Each row MUST start with `participant_id`.
+
+-   Each TSV file MUST contain a `session_id` column when
+    multiple [sessions](../glossary.md#session-entities)[<sup>1</sup>](#footnotes) are present
+    in the data set regardless of whether those sessions are in
+    the `phenotype/` data, `sub-<label>/` data, or a combination of the two.
+
+-   If more than one of the same measurement tool is acquired within
+    the same `session_id`, a `run_id` column MUST be added.
+
+-   Encoding the acquisition time for a measurement tool’s `session_id`,
+    is RECOMMENDED. This information MUST be stored in the `sessions.tsv`
+    file at the root level of the dataset in the `acq_time` column.
```

Comment:
@effigies mentioned this as "This is logically equivalent to "the acq_time column MUST NOT appear in a phenotype TSV file", but it takes some thinking about to get there. The spec should just say that."

I agree with the explicit "MUST NOT", But it also goes a step further in enforcing a root level sessions.tsv.